README.md

```diff
@@ -44,11 +44,17 @@ You need to install autoconf, automake and libtool:
 
 	brew install autoconf automake libtool
 	
+In case OpenSSL is not installed:
+    
+	brew install openssl
+	cd /usr/local/include
+	ln -s ../opt/openssl/include/openssl .
```

Comment:
Most out there recommend to use the soft link solution, I agree it's not the most elegant, but macOS has some issues code compliance... Basically Apple removed support for OpenSSL on OS X El Captain (after the famous OpenSSL bug). See e.g. this proposed solution, which is frequently cited: https://solitum.net/openssl-os-x-el-capitan-and-brew/